bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerStorage.java

```diff
@@ -103,12 +104,19 @@ void initialize(ServerConfiguration conf,
     long addEntry(ByteBuf entry) throws IOException, BookieException;
 
     /**
-     * Read an entry from storage.
+     * Read an entry from storage. In case of {@link BookieProtocol#LAST_ADD_CONFIRMED} we are going to return the
```

Comment:
I am adding this comments because I had to dig into code to understand clearly what is the contract of the method